tutorials/R/R-skills/Colorado-floods-data-visualization/USGS-Stream-Discharge-In-R/USGS-Stream-Discharge-In-R.md

```diff
@@ -365,7 +377,7 @@ October 15 2013.
     start.end <- c(startTime,endTime)
     start.end
 
-    ## [1] "2013-08-15 MDT" "2013-10-15 MDT"
+    ## [1] "2013-08-15 EDT" "2013-10-15 EDT"
```

Comment:
This must have changed the time zone based on your computer. Any way around this? MDT would be appropriate for these dates in Colorado.